hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFileReader.java

```diff
@@ -251,11 +256,12 @@ private HoodieLogBlock readBlock() throws IOException {
           return HoodieAvroDataBlock.getBlock(content, readerSchema);
         } else {
           return new HoodieAvroDataBlock(logFile, inputStream, Option.ofNullable(content), readBlockLazily,
-              contentPosition, contentLength, blockEndPos, readerSchema, header, footer);
+              contentPosition, contentLength, blockEndPos, readerSchema, header, footer, this.keyField);
         }
       case HFILE_DATA_BLOCK:
         return new HoodieHFileDataBlock(logFile, inputStream, Option.ofNullable(content), readBlockLazily,
-              contentPosition, contentLength, blockEndPos, readerSchema, header, footer, enableInlineReading);
+            contentPosition, contentLength, blockEndPos, readerSchema,
+            header, footer, enableInlineReading, this.keyField);
```

Comment:
to be consistent with other arguments, "this" can be dropped for keyField.

Reply:
fixed.